// @aws-sdk/middleware-bucket-endpoint/dist-cjs/index.js
'use strict';

var s3 = require('@aws-sdk/middleware-sdk-s3/s3');



exports.NODE_DISABLE_MULTIREGION_ACCESS_POINT_CONFIG_OPTIONS = s3.NODE_DISABLE_MULTIREGION_ACCESS_POINT_CONFIG_OPTIONS;
exports.NODE_DISABLE_MULTIREGION_ACCESS_POINT_ENV_NAME = s3.NODE_DISABLE_MULTIREGION_ACCESS_POINT_ENV_NAME;
exports.NODE_DISABLE_MULTIREGION_ACCESS_POINT_INI_NAME = s3.NODE_DISABLE_MULTIREGION_ACCESS_POINT_INI_NAME;
exports.NODE_USE_ARN_REGION_CONFIG_OPTIONS = s3.NODE_USE_ARN_REGION_CONFIG_OPTIONS;
exports.NODE_USE_ARN_REGION_ENV_NAME = s3.NODE_USE_ARN_REGION_ENV_NAME;
exports.NODE_USE_ARN_REGION_INI_NAME = s3.NODE_USE_ARN_REGION_INI_NAME;
exports.bucketEndpointMiddleware = s3.bucketEndpointMiddleware;
exports.bucketEndpointMiddlewareOptions = s3.bucketEndpointMiddlewareOptions;
exports.bucketHostname = s3.bucketHostname;
exports.getArnResources = s3.getArnResources;
exports.getBucketEndpointPlugin = s3.getBucketEndpointPlugin;
exports.getSuffixForArnEndpoint = s3.getSuffixForArnEndpoint;
exports.resolveBucketEndpointConfig = s3.resolveBucketEndpointConfig;
exports.validateAccountId = s3.validateAccountId;
exports.validateDNSHostLabel = s3.validateDNSHostLabel;
exports.validateNoDualstack = s3.validateNoDualstack;
exports.validateNoFIPS = s3.validateNoFIPS;
exports.validateOutpostService = s3.validateOutpostService;
exports.validatePartition = s3.validatePartition;
exports.validateRegion = s3.validateRegion;
